File: app/src/main/java/com/valdroide/thesportsbillboardinstitution/main_user/fragment/fixture/ui/adapter/OnItemClickListener.java
package com.valdroide.thesportsbillboardinstitution.main_user.fragment.fixture.ui.adapter;

import com.valdroide.thesportsbillboardinstitution.model.entities.Fixture;

public interface OnItemClickListener {
    void onClickFixture(int position, Fixture fixture);
    void onClickButtonAddMore();
}

File: app/src/main/java/com/valdroide/thesportsbillboardinstitution/main_user/fragment/fixture/ui/adapter/OnItemClickListener.kt
package com.valdroide.thesportsbillboardinstitution.main_user.fragment.fixture.ui.adapter

import com.valdroide.thesportsbillboardinstitution.model.entities.Fixture

interface OnItemClickListener {
    fun onClickFixture(position: Int, fixture: Fixture)
    fun onClickButtonAddMore()
}
